yktrace/src/lib.rs

```diff
@@ -50,22 +50,30 @@ impl IRBlock {
 
 /// An LLVM IR trace.
 pub struct IRTrace {
-    blocks: Vec<IRBlock>,
+    // The blocks of the trace. A None element represents an arbitrarily sized portion of unknown
+    // code which we were unable to determine LLVM IR for, e.g. external library code. The length
+    // of the unknown code is unknown.
```

Comment:
I think we've established the code is unknown so we can drop one of the "unknowns".